src/odemis/gui/cont/tabs.py

```diff
@@ -2039,6 +2078,20 @@ def __init__(self, name, button, panel, main_frame, main_data):
         panel.stage_align_btn_m_aligner_z.Bind(wx.EVT_BUTTON, self._on_aligner_btn)
         panel.btn_cancel.Bind(wx.EVT_BUTTON, self._on_cancel)
 
+    def load_data(self, streams):
+        """
+        Load the overview view with the given list of acquired static streams
+        :param streams: (list of StaticStream) the newly acquired static streams from the localization tab
+        """
+        # Replace the old streams with the newly acquired ones in the view
+        overview_view = self._tab_data_model.focussedView.value
+        existing_streams = overview_view.getStreams()
+        for stream in streams:
+            ex_st = next((ex_st for ex_st in existing_streams if ex_st.acquisitionType.value == stream.acquisitionType.value and ex_st.name.value == stream.name.value), None)
+            if ex_st:
+                overview_view.removeStream(ex_st)
```

Comment:
See my comments on the "clear_data" PR. I think we could simplify by checking just for the name.